Reference/Config/index.md

```diff
@@ -1,10 +1,10 @@
 #Configuration Files
 
-_This section will define the many configuration options in Umbraco and what they do_
+_This section will explain the many configuration options in Umbraco and what they do_
 
 ###[web.config](webconfig/index.md)
 
-There are quite a few umbraco options in the web.config file defined in appSettings. Click on your version below to view details:
+There are quite a few Umbraco options in the `web.config` file defined in `<appSettings>`. Click on your version below to view details:
```

Comment:
Tbh I have no idea. Looks like it is left overs from an older version of docs where there were version specific docs for web.config. Think it slipped through the cracks because there actually is something listed below.

Think that line should just go.